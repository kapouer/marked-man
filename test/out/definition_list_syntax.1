DEFITION_LIST_SYNTAX(5)                                DEFITION_LIST_SYNTAX(5)

NNAAMMEE
       ddeeffiittiioonn__lliisstt__ssyynnttaaxx - hiya

       Definition lists look like unordered lists:

              • term:
                definition

              • another one:
                The definition may span
                multiple lines and even
                start
                new paragraphs

              • ----ssoommeeaarrgg=<VALUE>:
                We can do that too.

                                 January 1979          DEFITION_LIST_SYNTAX(5)
