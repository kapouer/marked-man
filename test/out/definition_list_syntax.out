[4mDEFITION_LIST_SYNTAX[24m(5)       File Formats Manual      [4mDEFITION_LIST_SYNTAX[24m(5)

[1mNAME[0m
       [1mdefition_list_syntax [22m- hiya

       Definition lists look like unordered lists:

        â€¢ term:
          definition

        â€¢ another one:
          The definition may span
          multiple lines and even
          start
          new paragraphs

        â€¢ [1m--somearg[22m=<VALUE>:
          We can do that too.

                                 January 1979          [4mDEFITION_LIST_SYNTAX[24m(5)
